litmus-portal/frontend/src/components/InputField/styles.ts

```diff
@@ -78,7 +80,7 @@ const useStyles = makeStyles((theme) => ({
     borderColor: theme.palette.error.main,
   },
   inputArea: {
-    width: theme.spacing(45),
+    width: 'inherit',
```

Comment:
Inheriting the width will make the input fields go out of shape for Login, Settings and other screens where there were fixed width before. If we face a use case where we want bigger input fields we can pass the width as a style `style:{{width: <size>}}`

Reply:
I have used `inherit` here because there is a requirement of width change and it's defined in the component itself. So now when we use this component we can pass the width from the parent. but a/c to your approach can we change the width without changing in the components?

Reply:
App width is always fixed and it y default takes the entire screen size, we can define the width of any components we are using on top of it. By default it is `100%` I believe, by making it auto we are opening the possibility for it to change. I guess you can use width auto on any component that you are passing from the parent not on App itself.